### PR TITLE
Banner Manager UI - sidebar + stick bottom bar

### PR DESCRIPTION
**Pull request by TomPretty**
## What does this change?

Add the styling/functionality of the sidebar and sticky bottom bar. 

## Images
Screenshot 2020-08-07 at 09 02 24

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true,
+}
```
Comment (TomPretty) [inline .vscode/settings.json:3]: Should this be in the gitignore? I wasn't sure, so I've just left it here for now

Comment (tomrf1) [reply .vscode/settings.json:3]: yes - please add to gitignore, I think we should keep IDE-specific files out of here

Comment (TomPretty) [reply .vscode/settings.json]: nice - I've removed it now and update the gitignore

File: public/src/components/channelManagement/testPriorityLabelListLabel.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      padding: "4px 0",
+    },
+    text: {
+      fontSize: "12px",
+      color: palette.grey[700],
+    },
+    dashedLinesContainer: {
+      display: "flex",
+      marginLeft: "4px",
+      "& > * + *": {
```
Comment (paulbrown1982) [inline public/src/components/channelManagement/testPriorityLabelListLabel.tsx:24]: 😝

**Pull request merged**